### PR TITLE
Account price based on rem.oracle

### PR DESCRIPTION
**Pull request by Alladin9393**
## Consensus Changes
- [ ] Consensus Changes

## API Changes
- [ ] API Changes

## Documentation Additions
- [ ] Documentation Additions

### CODE REVIEW

File: contracts/contracts/rem.system/src/rem.system.cpp [hunk 1]
```diff
@@ -160,6 +160,14 @@ namespace eosiosystem {
       _gstate.min_account_stake = min_account_stake;
    }
 
+   uint64_t system_contract::get_min_account_stake() {
+      eosio::remprice_idx remprice_table(oracle_account, oracle_account.value);
+      auto it = remprice_table.find(rem_usd_pair.value);
+      bool is_valid_price = (it != remprice_table.end()) && ((current_time_point() - it->last_update.to_time_point()) <= eosio::minutes(70));
```
Comment (roman-tik) [inline contracts/contracts/rem.system/src/rem.system.cpp]: 70 minutes should be parametrized, let's avoid magic numbers

File: contracts/contracts/rem.system/src/rem.system.cpp [hunk 2]
```diff
@@ -160,6 +160,14 @@ namespace eosiosystem {
...
+
+      return is_valid_price ? std::min(uint64_t(min_account_price / it->price), _gstate.min_account_stake) : _gstate.min_account_stake;
```
Comment (roman-tik) [inline contracts/contracts/rem.system/src/rem.system.cpp]: make sure `uint64_t(min_account_price / it->price) != 0`

**Review by roman-tik — CHANGES_REQUESTED** [review]

File: contracts/contracts/rem.system/src/rem.system.cpp (hunk 2 above)
Comment (Alladin9393) [reply contracts/contracts/rem.system/src/rem.system.cpp]: @roman-tik, done.

File: contracts/contracts/rem.system/src/rem.system.cpp (hunk 1 above)
Comment (Alladin9393) [reply contracts/contracts/rem.system/src/rem.system.cpp]: @roman-tik, done.

**Review by roman-tik — APPROVED** [review]

**Pull request merged**